rosetta-lang/src/main/java/com/regnosys/rosetta/generator/java/object/MetaFieldGenerator.xtend

```diff
@@ -71,34 +73,33 @@ class MetaFieldGenerator {
 //			}
 //		}
 		
-		val modelClasses = model.elements.filter [
-			it instanceof Data
-		]
-		if (modelClasses.empty) {
-			return
-		}
 		
 		//find all the reference types
-		val namespaceClasses = Multimaps.index(modelClasses, [c|c.model]).asMap
+		val namespaceClasses = Multimaps.index(model.elements, [c|c.model]).asMap
 		for (nsc : namespaceClasses.entrySet) {
```

Comment:
This doesn't make sense. We have a `model`. We get all their elements `model.elements`. We then index them by their model `Multimaps.index(model.elements, [c|c.model])`.

Instead of doing `for (nsc : namespaceClasses.entrySet)`, directly do:
```
for (attr : model.eAllOfType(Attribute).filter[RMetaAnnotatedType.hasMeta])
```

Reply:
done